kong/plugins/prometheus/exporter.lua

```diff
@@ -31,6 +31,58 @@ package.loaded['prometheus_resty_counter'] = require("resty.counter")
 local kong_subsystem = ngx.config.subsystem
 local http_subsystem = kong_subsystem == "http"
 
+
+-- should we introduce a way to know if a plugin is configured or not?
+local is_prometheus_enabled, register_event do
+  local plugin_name = "prometheus"
```

Comment:
constant `plugin_name` `cache_key` should be upper case.

Reply:
done